docker/2diirunner-with-packages/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM maurolepore/2diirunner:1.3.0
```

Comment:
This Dockerfile will trigger a build on dockerhub with every push -- now configured only to the branch `docker`, but eventually it should be configured with `master`.

Reply:
Likely `FROM maurolepore/2diirunner:1.3.0` should be `FROM maurolepore/2diirunner:latest`, so we don't have to rewrite this Dockerfile when transitionmonitor increment the version number.